### PR TITLE
Add partnership details to check your answers page.

### PR DESCRIPTION
**Pull request by reggieb**
See [RIP-94](https://eaflood.atlassian.net/browse/RIP-94) for details

### CODE REVIEW

**Comment by woodpigeon** [comment]
Typo in PR title ;-)

**Comment by reggieb** [comment]
Doh! I've been doing that all the way through building the partnership stuff.

File: app/presenters/flood_risk_engine/address_presenter.rb [hunk 1]
```diff
@@ -9,9 +9,7 @@ def initialize(address)
     # Flat 1A, 121 Acacia Avenue, Broadbean, Kent, TU4 8AA
     def to_single_line
       return unless address
-      %i(premises street_address locality city postcode).map do |attribute|
-        address[attribute]
-      end.compact.reject(&:blank?).join(", ")
+      address.parts.join(", ")
     end
```
Comment (woodpigeon) [inline app/presenters/flood_risk_engine/address_presenter.rb]: Some elements could be blank - how about
`address.parts.reject(&:blank?).join(", ")`

File: app/presenters/flood_risk_engine/address_presenter.rb [hunk 2]
```diff
@@ -9,9 +9,7 @@ def initialize(address)
     # Flat 1A, 121 Acacia Avenue, Broadbean, Kent, TU4 8AA
     def to_single_line
       return unless address
-      %i(premises street_address locality city postcode).map do |attribute|
-        address[attribute]
-      end.compact.reject(&:blank?).join(", ")
+      address.parts.join(", ")
```
Comment (woodpigeon) [inline app/presenters/flood_risk_engine/address_presenter.rb]: Some elements could be blank - how about
`address.parts.reject(&:blank?).join(", ")`

File: app/presenters/flood_risk_engine/partner_presenter.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    attr_reader :partner
+    delegate :full_name, :address, to: :partner
+
+    def initialize(partner)
+      @partner = partner
+    end
+
+    def to_single_line
+      name_and_address_parts.join(", ")
+    end
```
Comment (woodpigeon) [inline app/presenters/flood_risk_engine/partner_presenter.rb]: could be blank elements leading to ", , "

File: app/presenters/flood_risk_engine/address_presenter.rb (hunk 2 above)
Comment (reggieb) [inline app/presenters/flood_risk_engine/address_presenter.rb]: The `address#parts` method already includes a `.reject(&:blank?)`. Have a look at the `Address` model

**Comment by woodpigeon** [comment]
:ship: away!

**Pull request merged**